src/components/dynamic/form/Judge.jsx

```diff
@@ -15,6 +15,7 @@ const Judge = () => {
     name: session.user.name,
     email: session.user.email,
     roles: session.user.roles,
+    ...(session.user.photo && { photo: session.user.photo }),
```

Comment:
if u just add the photo: session.user.photo it should work regardless? you can simplify it to something like:
```jsx
photo: session.user.photo ?? null 
```
or something similar